docs/intro/channels.txt

```diff
@@ -0,0 +1,698 @@
+Getting Started with Channels
+=============================
+
+Now, let's get to writing some consumers. If you've not read it already,
```

Comment:
I think this transition/intro could be better. Maybe we could call the doc something like: "Advanced tutorial: Intro to channels" and then give a better overview about the goals of this tutorial.